doc/howto/cross-network-inet-multicast-howto.md

```diff
@@ -0,0 +1,128 @@
+# OpenWeave + Happy Cross Network Multicast Inet Layer HOWTO
+
+You have decided that you would like to experiment with cross-network multicast using OpenWeave. This how-to guide illustrates using the Happy network simulation tool to set up a virtual topology that demonstrates using OpenWeave’s Inet layer across two distinct networks with a multicast proxy.
+
+## Step 1: Download and Build mcproxy
+
+The IPv6 multicast proxy daemon, _mcproxy_, serves to proxy / forward / route IPv6 multicast traffic across two disjoint IPv6 network links.
+
+```
+% git clone https://github.com/mcproxy/mcproxy.git mcproxy
+% sudo apt-get install qt5-qmake qt5-default
+% cd mcproxy/mcproxy
+% qmake
+% make
+```
+
+## Step 2: Download, Build, and Install happy
+
+```
+% git clone https://github.com:openweave/happy.git happy
```

Comment:
I get an error running it this way.  Should it be 'git clone https://github.com/openweave/happy.git happy'?

Reply:
Good catch, thanks. Will fix.